modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock.adoc

```diff
@@ -77,10 +86,29 @@ $ oc logs linuxptp-daemon-74m2g -n openshift-ptp -c linuxptp-daemon-container
 .Example output
 [source,terminal]
 ----
-ts2phc[94980.334]: [ts2phc.0.config] nmea delay: 98690975 ns
-ts2phc[94980.334]: [ts2phc.0.config] ens3f0 extts index 0 at 1676577329.999999999 corr 0 src 1676577330.901342528 diff -1
-ts2phc[94980.334]: [ts2phc.0.config] ens3f0 master offset         -1 s2 freq      -1
-ts2phc[94980.441]: [ts2phc.0.config] nmea sentence: GNRMC,195453.00,A,4233.24427,N,07126.64420,W,0.008,,160223,,,A,V
-phc2sys[94980.450]: [ptp4l.0.config] CLOCK_REALTIME phc offset       943 s2 freq  -89604 delay    504
-phc2sys[94980.512]: [ptp4l.0.config] CLOCK_REALTIME phc offset      1000 s2 freq  -89264 delay    474
+ptp4l[69011.473]: [ptp4l.1.config] master offset          1 s2 freq   -7272 path delay       528
+I0816 13:51:13.521655 1781035 pmc.go:27] pmc read config from /var/run/ptp4l.1.config
+I0816 13:51:13.521742 1781035 pmc.go:28] pmc run command: GET PARENT_DATA_SET
+ptp4l[69011.474]: [ptp4l.0.config] master offset          1 s2 freq  -13442 path delay       528
+I0816 13:51:13.521974 1781035 pmc.go:27] pmc read config from /var/run/ptp4l.0.config
+I0816 13:51:13.521995 1781035 pmc.go:28] pmc run command: GET PARENT_DATA_SET
+I0816 13:51:13.524024 1781035 pmc.go:40] pmc result: sending: GET PARENT_DATA_SET
```

Comment:
wrong output, ptp4l should not be writing offset for GM 
Also remove  PMC command output , This  is just noise will go away . What you had before seems to be correct